internal/source/argocd/default-config.yaml

```diff
@@ -0,0 +1,364 @@
+log:
+  level: "info"
+
+interactivity:
```

Comment:
Maybe it is better add a field description, "Not all the platforms support this. Only Slack for now". Or something relevant.